Oleksandr_Kotov/6/decorators.py

```diff
@@ -0,0 +1,51 @@
+"""module provides decorators for the game
+"""
+
+import game_logger
+
+
+def info_decorator(func):
+    """decorate a function into info wrapper
+
+    Arguments:
+        func callable -- function to wrap
+
+    Returns:
+        callable -- wrapped function
+    """
+
+    def wrapper(*args, **kwargs):
+
+        return func(*args, **kwargs)
+
+    return wrapper
+
+
+def debug_decorator(func):
+    """decorates a function into debug wrapper
+    debug wrapper prints function name and arguments before call
+
+    Arguments:
+        func callable -- function to wrap
+
+    Returns:
+        callable -- wrapped function
+    """
+
+    def wrapper(*args, **kwargs):
+
+        from main_loop import debug_mode
+
+        if debug_mode:
+
```

Comment:
You should use @functools.wraps for original function introspection.

Reply:
got it